specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2018-01-01/examples/Events_ListBySubscriptionId.json

```diff
@@ -0,0 +1,76 @@
+﻿{
+	"parameters": {
+		"subscriptionId": "subscriptionId",
+		"api-version": "2018-01-01",
+		"$filter": "service eq 'Virtual Machines' or region eq 'West US'"
+	},
+	"responses": {
+		"200": {
+			"body": {
+				"value": [
+					{
+						"id": "/providers/Microsoft.ResourceHealth/events/VIRTUALMACHINES-WESTUS-DTR1",
+						"name": "VIRTUALMACHINES-WESTUS-DTR1",
+						"type": "/providers/Microsoft.ResourceHealth/events",
+						"properties": {
+							"eventType": "serviceissue",
```

Comment:
The allowed values for `eventType` and `eventSource` have a different spelling than specified here - please adjust either the enum definitions or the example (to match your actual service as it is deployed right now)

Reply:
Changed the examples to use correct values from enum.